### ISSUE TITLE
No support for cameras?

### ISSUE THREAD

**Issue by jonhakr** [ndg63276/smartlife#10]
I seem to have logged inn successfully, but It does not display anything.
I only see the buttons Theme, Autorefresh, Refresh and Logout.

Is this because there is no support for security cameras (I only have these on my Smart Life account)?

Regards,
JH

**Comment by ndg63276** [comment]
Yes, pretty much, I don't have any cameras to test it on. What would you expect to be able to do with a camera?

**Comment by jonhakr** [comment]
Main reason would be to be able to view the video stream on a PC in real time for surveilance, rather than the app on a phone.
But I assume setting up a video stream etc is quite different from using their API to configure devices and trigger events (and might not even be possible?)

**Comment by ndg63276** [comment]
Yeah, that would be my fear.

If you are technical, you could open the dev console in your browser (F12 or Ctrl-Shift-i) and type:
```
user_info.devices
```
That is all the info the website gets from the API. You could see if any of the items has a URL for the video or anything.

**Comment by jonhakr** [comment]
Doesn't seem like there is any devices or info listed, but thanks anyway!
